src/particleContainer/AutoPasContainer.cpp

```diff
@@ -64,33 +66,38 @@ auto parseAutoPasOption(XMLfileUnits &xmlconfig, const std::string &xmlString, c
 void AutoPasContainer::readXML(XMLfileUnits &xmlconfig) {
 	string oldPath(xmlconfig.getcurrentnodepath());
 
-	// set default values here!
-
-	_traversalChoices = parseAutoPasOption<autopas::TraversalOption>(xmlconfig, "allowedTraversals", "c08");
-
-	_containerChoices = parseAutoPasOption<autopas::ContainerOption>(xmlconfig, "allowedContainers", "linked-cell");
-
+	// if any option is not specified in the XML use the autopas defaults
+	// get option values from xml
+	_traversalChoices = parseAutoPasOption<autopas::TraversalOption>(xmlconfig,
+	                                                                 "allowedTraversals",
+	                                                                 _autopasContainer.getAllowedTraversals());
+	_containerChoices = parseAutoPasOption<autopas::ContainerOption>(xmlconfig,
+	                                                                 "allowedContainers",
+	                                                                 _autopasContainer.getAllowedContainers());
```

Comment:
provide `getMostContainers`

Reply:
instead of specifying what to use here use AutoPas' defaults. Since [autopas #471](https://github.com/AutoPas/AutoPas/pull/471) these are not all but "most"